apps/mobile-alphabet/src/app/Components/Credits.tsx

```diff
@@ -1,10 +1,14 @@
+import { colors } from 'app/styles';
 import { Text, View } from 'react-native';
+import Background from './Background';
 
 export function CreditsScreen() {
     return (
-        <View>
-            <Text>Credits screen works!</Text>
-        </View>
+        <Background>
+            <View>
+                <Text style={{ color: colors.text }}>Credits screen works!</Text>
```

Comment:
Can you remove this and make adding credits part of a separate PR? When you do that, keep in mind that the credits should come from the back-end.

Reply:
The Credits placeholder has been there for a while. I added the Background wrapper here.